src/components/composer/composer/composer.ts

```diff
@@ -370,7 +379,7 @@ export class Composer extends Component<Props, SpreadsheetChildEnv> {
 
   showAutocomplete(searchTerm: string) {
     this.autoCompleteState.showProvider = true;
-    let values = Object.entries(functionRegistry.content).map(([text, { description }]) => {
+    let values = Object.entries(this.nonHiddenFunctions).map(([text, { description }]) => {
```

Comment:
what about just adding `.filter(([text, { hidden }]) => !hidden)` ?
It look simpler than unpack, filter, reduce
```suggestion
    let values = Object.entries(functionRegistry.content).filter(([text, { hidden }]) => !hidden).map(([text, { description }]) => {
```